docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/java/custom-tags-configuration.md

```diff
@@ -14,6 +14,8 @@ OTEL_RESOURCE_ATTRIBUTES="service.name=Accounts"
 OTEL_RESOURCE_ATTRIBUTES="application=OnlineBanking,exampleKey=exampleValue"
 ```
 
+{@import ../../../../../reuse/otel-deployment-environment-tag.md}
```

Comment:
Please add definition to `Custom tags configuration via startup parameter` paragraph